python/lsst/obs/subaru/instrument.py

```diff
@@ -45,6 +46,22 @@
 log = logging.getLogger(__name__)
 
 
+class _HscAbstractFilterKeyHandler(KeyHandler):
+    """KeyHandler for HSC filter keys that should be mapped to AbstractFilters.
```

Comment:
Is this materially different from `PhysicalToAbstractFilterKeyHandler`? I don't remember why I kept it around when I was doing the original converter cleanup (if there was a reason beyond that I just missed it), but I think it can just be replaced by that.

Reply:
Yup, looks like they're identical.  Will drop this one.